bootbox.js

```diff
@@ -317,9 +319,15 @@
 
   exports.confirm = function() {
     var options;
+    var invertButtons = defaults.invertButtons;
+    var dialog;
 
     options = mergeDialogOptions("confirm", ["cancel", "confirm"], ["message", "callback"], arguments);
```

Comment:
RE my main comment about swapping the button order, we could easily just swap the array passed as the second param here...